ingest/expression_files/dense_ingestor.py

```diff
@@ -128,14 +132,22 @@ def is_r_formatted_file(header, row):
             header (List[str]): Header of the dense matrix
             row (List): A single row from the dense matrix
         """
-        # An "R formatted" file has one less entry in the header
-        # row than each successive row. Also, "GENE" will not appear in header
+        # An "R formatted" file can:
+        # Not have gene in the header or
+        # Have one less entry in the header than each successive row or
+        # Have "" as the last value in header.
         if header[0].upper() != "GENE":
             length_of_next_line = len(row)
```

Comment:
I think 'of' is typically disfavored in variable names.  I would rename to `next_line_length`